### ISSUE TITLE
v2.x failures on travis

### ISSUE THREAD

**Issue by tacaswell** [matplotlib/matplotlib#6120]
```python
======================================================================
FAIL: matplotlib.tests.test_axes.test_log_scales.test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/matplotlib/matplotlib/venv/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/matplotlib/matplotlib/venv/lib/python3.5/site-packages/matplotlib-1.5.1+260.gebfbc4a-py3.5-linux-x86_64.egg/matplotlib/testing/decorators.py", line 54, in failer
    result = f(*args, **kwargs)
  File "/home/travis/build/matplotlib/matplotlib/venv/lib/python3.5/site-packages/matplotlib-1.5.1+260.gebfbc4a-py3.5-linux-x86_64.egg/matplotlib/testing/decorators.py", line 245, in do_test
    '(RMS %(rms).3f)'%err)
matplotlib.testing.exceptions.ImageComparisonFailure: images not close: /home/travis/build/matplotlib/matplotlib/result_images/test_axes/log_scales_svg.png vs. /home/travis/build/matplotlib/matplotlib/result_images/test_axes/log_scales-expected_svg.png (RMS 0.009)
----------------------------------------------------------------------
```

I can not reproduce this locally, can anyone else?

**Comment by WeatherGod** [comment]
That is a tiny, tiny diff, and in the SVG, too. Could it be a change in
inkscape?

**Comment by mdboom** [comment]
Yeah -- I suspect the reason we can't reproduce locally is most of us have a newer version of Inkscape than on Travis.  The differences -- particularly when looking at the content of the SVG itself -- are all inconsequential.  I think the best fix is probably just replace the SVG with the new output.

**Comment by jenshnielsen** [comment]
Fixed by #6127
